crates/test_utils/src/main.rs

```diff
@@ -3,10 +3,10 @@ use std::process::{exit, Command};
 use test_utils::newman_runner;
 
 fn main() {
-    let mut newman_command: Command = newman_runner::command_generate();
+    let mut runner: (Command, bool, String) = newman_runner::command_generate();
```

Comment:
We'll need to either destructure the tuple returned and have meaningful variable names, or return a struct with named fields so that it's easy to read and understand what's being returned.